content/docs/examples/example-scripts.md

```diff
@@ -438,6 +438,18 @@ publish("example")
   .postOps(ctx => `grant select on ${ctx.self()} to role`)
 ```
 
+### Creating incremental tables using the JS API
+
+```js
+// definitions/incremental_example.js
+
+publish("incremental_example", {
+  type: "incremental"
+}).query(ctx =>
+  `SELECT * FROM ${ctx.ref("other_table")}
+${ctx.when(ctx.incremental(),`where timestamp > (select max(date) from ${ctx.self()}`)}`)
```

Comment:
```suggestion
}).query(ctx => `
  SELECT * FROM ${ctx.ref("other_table")}
  ${ctx.when(ctx.incremental(),`WHERE timestamp > (SELECT MAX(date) FROM ${ctx.self()}`)}
`)
```

(slightly nicer spacing IMO, also i don't mind if we use CAPITAL_CASE or lower_case but we should be consistent)